lib/galaxy/metadata/set_metadata.py

```diff
@@ -15,6 +15,12 @@
 import os
 import sys
 import traceback
+import warnings
+
+# Warning on stderr will cause some jobs to be failed.
+# This silences deprecation warnings in the cryptography module.
+if sys.version_info.major <= 3 and sys.version_info.minor < 6:
```

Comment:
Should we make this unconditional, i.e. remove this line?

Reply:
We could, but this isn't the right fix, unfortunately.